funnel/views/project.py

```diff
@@ -167,6 +170,20 @@ def edit(self):
                 else:
                     return render_form(form=form, title=_("Edit project"), submit=_("Save changes"), autosave=True)
 
+    @route('cfp', methods=['GET', 'POST'])
+    @lastuser.requires_login
+    @requires_permission('edit_project')
+    def cfp(self):
+        form = CFPForm(obj=self.obj, parent=self.obj.profile, model=Project)
```

Comment:
`parent` is not required here as there is no `AvailableName` validator in this form.